app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeSheet.kt

```diff
@@ -39,11 +45,39 @@ class ConfirmFacilityChangeSheet : BottomSheetActivity() {
     positiveButtonClicks()
   }
 
+  private val delegate by unsafeLazy {
+    MobiusDelegate.forActivity(
+        events.ofType(),
+        ConfirmFacilityChangeModel(),
+        ConfirmFacilityChangeUpdate(),
+        effectHandlerFactory.create(this).build(),
+        ConfirmFacilityChangeInit(),
+        { /* No-op, there's nothing to render */ }
```

Comment:
We don't have to provide no-op for `modelUpdateListener` in Mobius loop anymore. `MobiusDelegate.forActivity` has default parameter for `modelUpdateListener` so you can remove the no-op.

Reply:
Did you forget to remove the no-op for `modelUpdateListener` here? 😅